setup/lang/gwb2ged.htm

```diff
@@ -19,10 +14,11 @@
 lv: Izveidot GEDCOM datni
 sv: Exportera en GEDCOM-fil
 ]</title>
+  <meta name="robots" content="none">
+  <meta charset="UTF-8">
```

Comment:
If this is utf-8, maybe the document may use accent instead of escape `&eacute;` ?